keyboards/zen/rev1/config.h

```diff
@@ -54,8 +50,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
-#define BACKLIGHT_PIN F4
-#define BACKLIGHT_LEVELS 5
+/* key combination for command */
+#define IS_COMMAND() ( \
```

Comment:
Could you remove this?  

This is the default, and shouldn't be needed.

Reply:
Removed